### DISCUSSION TITLE
Google Fonts with NextJS

### DISCUSSION THREAD

**Discussion by filippobrigati** [vercel/next.js#35719] [Help]
Hello everyone, I'm learning next js but I have a problem... I tried to add a different font using @import from google fonts and everything works fine, but when I add a getServerSideProps() function the font doesn't work while with getStaticProps() everything works fine and I can't understand the problem. [This](https://github.com/filippo-brigati/filippobrigati) is the repo. Thanks

**Comment by bartekczyz** [comment]
Same problem here

**Reply by filippobrigati** [reply]
Hi, I fixed using this method which you can find in my portfolio repository --> [here](https://github.com/filippo-brigati/filippobrigati)
